projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php

```diff
@@ -266,7 +266,14 @@ public function maybe_logout_user_from_wpcom( $redirect_to ) {
 	 * @return string
 	 */
 	public function admin_body_class( $admin_body_classes ) {
-		return "$admin_body_classes jetpack-masterbar";
+
+		$classes = array( 'jetpack-masterbar' );
+
+		if ( get_option( 'wpcom_admin_interface' ) === 'wp-admin' ) {
+			$classes[] = 'wpcom-admin-interface';
+		}
+
+		return implode( ' ', $classes ) . $admin_body_classes;
```

Comment:
I believe we need a space here:

```suggestion
		return implode( ' ', $classes ) . ' ' . $admin_body_classes;
```

Otherwise the `$admin_body_classes` variable will be concatenated together and produce an incorrect class name.

Another alternative would be to reduce this line to `implode(' ', $classes)` and add `$admin_body_classes` to the `$classes` array so that it's imploded separated by the space.

Reply:
Thanks, thats a good point. When i tested it, the class names were separated by spaces and it worked as expected but i've changed the code just to be sure and make it more explicit.